src/FrameBuilder.php

```diff
@@ -84,7 +84,7 @@ public function buildFromBacktraceFrame(string $file, int $line, array $backtrac
             $strippedFilePath,
             $line,
             $rawFunctionName,
-            Frame::INTERNAL_FRAME_FILENAME !== $file && $strippedFilePath !== $file ? $file : null,
+            Frame::INTERNAL_FRAME_FILENAME !== $file ? $file : null,
```

Comment:
I was also thinking about moving this check to `Frame` constructor.

Reply:
I prefer to leave the responsibility of deciding whether the `$absoluteFilePath` argument should be filled or not to the builder